package br.com.digitalhouse.desafio.exceptions

class CursoExistenteException : Exception()